doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md

```diff
@@ -5,4 +5,107 @@ permalink: /:path/
 sort_idx: 20
 ---
 
-SDK integration instructions for select platforms
+# The guide provides information on how to cross-compile C++ SDK for Raspberry PI
+
+**Verified against:**
+
+**Host OS:** Ubuntu 14.04 LTS Desktop 64-bit.
+
+**The further instructions must be executed on the host machine**
+
+1. Download and install toolchain.
+
+        mkdir rpi_root && cd rpi_root
+        export RPI_ROOT=$(pwd)
+        git clone https://github.com/raspberrypi/tools.git
+        export INSTALLDIR=$RPI_ROOT/tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian
+        export PATH=$PATH:$INSTALLDIR/bin
+        export CROSS=arm-linux-gnueabihf
+        export CC=${CROSS}-gcc
+        export LD=${CROSS}-ld
+        export AS=${CROSS}-as
+        export AR=${CROSS}-ar
+
+2. Download and install zlib.
+
+        cd $RPI_ROOT
+        wget http://zlib.net/zlib-1.2.8.tar.gz
+        tar -xvzf zlib-1.2.8.tar.gz
+        cd zlib-1.2.8
+        ./configure --prefix=${INSTALLDIR}/arm-linux-gnueabihf/libc/usr
+        make && make install
+
+3. Download and install bzib2.
+
+        cd $RPI_ROOT
+        wget http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz
+        tar -xvzf bzip2-1.0.6.tar.gz
+        cd bzip2-1.0.6
+        sed -e "/^all:/s/ test//" -i Makefile
+        make CC="${CC}" AR="${AR}"
+        make PREFIX=${INSTALLDIR}/arm-linux-gnueabihf/libc/usr install
+
+4. Download and install boost.
+
+        cd $RPI_ROOT
+        wget -O boost_1_59_0.tar.bz2 http://sourceforge.net/projects/boost/files/boost/1.59.0/boost_1_59_0.tar.bz2/download
+        tar -xvjf boost_1_59_0.tar.bz2
+        cd boost_1_59_0
+        ./bootstrap.sh
+
+    Edit the `project-config.jam` file. Add 'using gcc : arm : arm-linux-gnueabihf-c++ ;' instead of  'using gcc ;':
```

Comment:
Add 'using gcc : arm : arm-linux-gnueabihf-c++ ;' instead of  'using gcc ;':

-->

 Add `using gcc : arm : arm-linux-gnueabihf-c++ ;` instead of  `using gcc ;`: